frontends/halo2/src/synthesis/regions/region_row.rs

```diff
@@ -46,33 +47,31 @@ impl<'r, 'io, 'fq, F: Field> RegionRow<'r, 'io, 'fq, F> {
         }
     }
 
-    fn enabled(&self) -> HashSet<&'r Selector> {
-        self.region
-            .selectors_enabled_for_row(self.row.row)
-            .into_iter()
-            .collect()
+    fn enabled(&self) -> Cow<SelectorSet> {
+        self.region.selectors_enabled_for_row(self.row.row)
     }
 
-    pub fn region_index(&self) -> Option<usize> {
-        self.region.index().map(|f| *f)
-    }
+    //pub fn region_index(&self) -> Option<usize> {
+    //    self.region.index().map(|f| *f)
+    //}
 
-    pub fn region_name(&self) -> &str {
-        self.region.name()
-    }
+    //pub fn region_name(&self) -> &str {
+    //    self.region.name()
+    //}
 
     pub fn row_number(&self) -> usize {
         self.row.row
     }
 
-    pub fn region_index_as_str(&self) -> String {
-        match self.region_index() {
-            Some(i) => i.to_string(),
-            None => "<unk>".to_string(),
-        }
-    }
+    //pub fn region_index_as_str(&self) -> String {
```

Comment:
Should we delete?

Reply:
Ah, yes. Guess I forgot to remove some dead code I commented out